README.rst

```diff
@@ -1024,12 +1029,9 @@ See also: `CONTRIBUTING.md <https://github.com/falconry/falcon/blob/master/CONTR
 Legal
 -----
 
-Copyright 2013-2020 by Individual and corporate contributors as
+Copyright 2013-2021 by Individual and corporate contributors as
 noted in the individual source files.
 
-Falcon image courtesy of `John
```

Comment:
I removed this notice since that Falcon image from Wikipedia seems to be no longer used in this repo.